pkg/plugin/packetparser/packetparser_linux.go

```diff
@@ -380,24 +379,18 @@ func (p *packetParser) endpointWatcherCallbackFn(obj interface{}) {
 	}
 
 	iface := event.Obj.(netlink.LinkAttrs)
-
 	ifaceKey := ifaceToKey(iface)
-	lockMapVal, _ := p.interfaceLockMap.LoadOrStore(ifaceKey, &sync.Mutex{})
```

Comment:
Why are you deleting this? The `interfaceLockMap` allows us to store a per interface lock and we can create/delete multiple qdisc in parallel. This is necessary (in place of a single lock) because large number of pods can come up at the same time, and we should start capturing packets as quickly as possible.

Reply:
I'm still not clear on this. Why is a per-interface mutex necessary for packetparser to handle concurrent create/delete qdisc operations? As far as I understand, operations on different interfaces shouldn't cause a data race.